### ISSUE TITLE
[Feb 2025] Accessibility Audit: Support admin user flows

### ISSUE THREAD

**Issue by DanielSass** [CDCgov/prime-simplereport#8514]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the support admin user flows. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/admin/*`

Use cases to test:
* view support admin menu page
* edit or verify org identity
* add org admin
* create new device
* manage devices
* org data access

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

### Additional context
See tickets from previous audits #5772, #6332

**Comment by emyl3** [comment]
Findings:
- `/admin` [dev.simplereport.gov-Support-admin-2025-02-26.json](https://github.com/user-attachments/files/18993628/dev.simplereport.gov-Support-admin-2025-02-26.json)

**Comment by emyl3** [comment]
Issues created:
- https://github.com/CDCgov/prime-simplereport/issues/8543

**Comment by emyl3** [comment]
This is ready for review
